build/musicbrainz-dev/scripts/createdb.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+eval $( perl -Mlocal::lib="${MUSICBRAINZ_PERL_LOCAL_LIB}" )
+
+FTP_MB=ftp://ftp.eu.metabrainz.org/pub/musicbrainz
+IMPORT="fullexport"
+FETCH_DUMPS=""
+WGET_OPTIONS=""
+
+read -d '' -r HELP <<HELP
+Usage: $0 [-wget-opts <options list>] [-sample] [-fetch] [MUSICBRAINZ_FTP_URL]
+
+Options:
+  -fetch      Fetch latest dump from MusicBrainz FTP
+  -sample     Load sample data instead of full data
+  -wget-opts  Pass additional space-separated options list (should be
+              a single argument, escape spaces if necessary) to wget
+
+Default MusicBrainz FTP URL: $FTP_MB
+HELP
+
+if [ $# -gt 4 ]; then
+    echo "$0: too many arguments"
+    echo "$HELP"
+    exit 1
+fi
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -wget-opts )
+            shift
+            WGET_OPTIONS=$1
+            ;;
+        -sample )
+            IMPORT="sample"
+            ;;
+        -fetch  )
+            FETCH_DUMPS="$1"
+            ;;
+        -*      )
+            echo "$0: unrecognized option '$1'"
+            echo "$HELP"
+            exit 1
+            ;;
+        *       )
+            FTP_MB="$1"
+            ;;
+    esac
+    shift
+done
+
+TMP_DIR=/media/dbdump/tmp
+
+case "$IMPORT" in
+    fullexport  )
+        DUMP_FILES=(
+        mbdump.tar.bz2
+        mbdump-cdstubs.tar.bz2
+        mbdump-cover-art-archive.tar.bz2
+        mbdump-derived.tar.bz2
+        mbdump-stats.tar.bz2
+        mbdump-wikidocs.tar.bz2
```

Comment:
```suggestion
          mbdump.tar.bz2
          mbdump-cdstubs.tar.bz2
          mbdump-cover-art-archive.tar.bz2
          mbdump-derived.tar.bz2
          mbdump-stats.tar.bz2
          mbdump-wikidocs.tar.bz2
```

Reply:
Again, it has to be fixed in original script first, we don’t want these scripts to differ by coding style.

Reply:
This script is being split (working on PR) to allow fetching search indexes anyway.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-docker/pull/125/commits/5047754d5b5a7880667e007a04182012cc0a9f6a